__init__.py

```
@@ -1865,11 +1841,10 @@ def _play_notify_expired(self, alert: Alert):
                 play_audio(to_play).wait(60)
             # TODO this depends on the length of the audio file
             time.sleep(1)
-        
+
         # reset volume
         if not to_play == "ocp":
```

Ruff

Use `to_play != "ocp"` instead of `not to_play == "ocp"`

Replace with `!=` operator

(SIM201)